vars/pipelineVegavisorManageNode.groovy

```diff
@@ -284,6 +296,22 @@ void call() {
                         }
                     }
                 }
+                post {
+                    success {
+                        catchError {
+                            script {
+                                alert.enableAlerts(silenceID: ALERT_SILENCE_ID, delay: 5)
+                            }
+                        }
+                    }
+                    unsuccessful {
+                        catchError {
+                            script {
+                                alert.enableAlerts(silenceID: ALERT_SILENCE_ID, delay: 0)
+                            }
+                        }
+                    }
+                }
```

Comment:
we could use the `always` condition for that: 
https://www.jenkins.io/doc/book/pipeline/syntax/

Reply:
There is a different delay for each option.